### PR TITLE
i2-low score

### LINKED ISSUE #34: Game Statistic - Lowest Total Score
Lowest sum of the winning and losing teams’ scores

Name: lowest_total_score
Return Value: Integer

### PR DESCRIPTION
**Pull request by ryandbarnett**
Adds lowest total score test and method.

This relies on i2 high score's PR so you will see those changes as well. Once i2 high score is merged you should no longer see those changes.

### CODE REVIEW

File: lib/game_teams_collection.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def initialize(path)
   end
 
   def from_csv(path)
+    return if !File.exist?(path)
```
Comment (dhalverson) [inline lib/game_teams_collection.rb:16]: I believe the ||= syntax will also do this, we can do some refactoring here if needed

**Review by dhalverson — APPROVED** [review]
Looks good, let's discuss the return !File statement to ensure understanding next meeting

**Comment by ryandbarnett** [comment]
I'm doing the file check to allow the stat tracker to initialize with empty arrays instead of having an error if a bad path is given to the from_csv method. Doing this allowed me to instantiate an empty stat tracker (meaning empty games, teams, and game_teams instance variables) in my test file. Once I had a working empty stat tracker I could stub stat tracker's games, teams, and game_teams instance variables with mock objects.

**Comment by ryandbarnett** [comment]
I'm happy to discuss more though!

**Pull request merged**